status-icon.js

```diff
@@ -6,20 +6,41 @@ import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
 function getIcon(state) {
 	switch (state) {
 		case 'success':
+		case '1':
+		case 1:
 			return 'tier1:check-circle';
 		case 'warning':
+		case '2':
+		case 2:
 			return 'tier1:alert';
 		default:
 			return 'tier1:close-circle';
 	}
 }
 
+function getMessage(state, success, failure, warning) {
+	switch (state) {
+		case 'success':
+		case '1':
+		case 1:
+			return success;
+		case 'warning':
+		case '2':
+		case 2:
+			return warning;
+		default:
+			return failure;
+	}
+}
+
 class StatusIcon extends RtlMixin(LitElement) {
 
 	static get properties() {
 		return {
 			state: { type: String, reflect: true },
-			message: { type: String, reflect: true }
+			successMessage: { type: String, reflect: true },
+			failureMessage: { type: String, reflect: true },
+			warningMessage: { type: String, reflect: true }
```

Comment:
HTML attribute names need to be case insensitive and typically we'd use hyphens here:
```suggestion
			successMessage: { type: String, reflect: true, attribute: 'success-message' },
			failureMessage: { type: String, reflect: true, attribute: 'failure-message' },
			warningMessage: { type: String, reflect: true, attribute: 'warning-message' }
```

More info on why here:
https://github.com/BrightspaceUI/guide/wiki/LitElement-Best-Practices-&-Gotchas#-do-use-correct-casing-for-attributes-and-properties